avocado/utils/service.py

```diff
@@ -118,8 +118,9 @@ def method(cmd_result):
             # If service is stopped, exit_status is also not zero.
             # So, we can't use exit_status to check result.
             output = cmd_result.stdout.lower()
```

Comment:
@Hao-Liu actually can I ask you to move this after the `if re.search(r"unrecognized")`? With this patch it's only being used there.